lib/welcomePage.dart

```diff
@@ -0,0 +1,80 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/cupertino.dart';
+import 'signup.dart';
+
+class Welcome extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text('Visit Africa'),
+        centerTitle: true,
+        backgroundColor: Color(0xffB15C1E),
+      ),
+      body: Container(
+        child: Stack(
+          children: <Widget>[
+            Image.asset(
+              "assets/cover.png",
+              width: 1100,
+              height: 560,
+              fit: BoxFit.cover,
+            ),
+            Padding(
+              padding: const EdgeInsets.only(top: 470.0, left: 30.0),
+              child: Text(
+                'Welcome',
```

Comment:
Aren't we supposed to have Welcome to **visit Africa** ? what changed your mind?